### DISCUSSION TITLE
Can't load replays after small bugfix on 20.04.2023

### DISCUSSION THREAD

**Discussion by gabriellimo** [fraxiinus/ReplayBook#258] [Troubleshooting]
**Describe the bug**
LiteDB ENSURE: slot length must be empty before use

**To Reproduce**
Trying to load files after the bugfix that happened around 10pm gmt-3 21.04.2023

![image](https://user-images.githubusercontent.com/12302651/233534523-41463209-814c-48b3-b24f-809f9bc69d03.png)

**Comment by fraxiinus** [comment]
Try clearing the replay cache and restarting ReplayBook

https://www.fraxiinus.dev/ReplayBook/troubleshooting/replays-not-loading/#replay-load-failure

Go to 「Settings」 (gear icon) -> 「Replays」 -> 「Clear cache」 button under "Replay Data Cache".
